trueos-handbook/README.md

```diff
@@ -102,6 +102,11 @@ using TrueOS and/or FreeBSD before following these instructions.
 
 ### Required Applications <a name="reaps"></a>
 
+.. note:: In case you are building the documentation from an OS other that
+   TrueOS, please note that the only supported `make` tool is (obviously)
+   the one provided with TrueOS (NetBSD make). So, please make sure you are
```

Comment:
> (obviously) the one provided with TrueOS (NetBSD make)

To avoid confusing the reader, we try not to include parenthetical asides:
"... the only supported `make` tool is **NetBSD make**. This is provided with TrueOS."